packages/native/src/components/WebView.tsx

```diff
@@ -90,6 +90,16 @@ const NativeWebView: React.FC<WebViewProps> = ({
     );
   };
 
+  const getFinalWidth = () => {
+    if (typeof style?.width === "number") {
+      return style.width;
+    } else if (typeof style?.width === "string" && style.width.includes("%")) {
+      return width * (Number(style.width.replace("%", "")) / 100);
```

Comment:
Can we move the definition of `width` into this function? I think it would make it a little easier to understand.

Reply:
@adnelson done.